sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample12_ConsumeEventsWithEventProcessor.cs

```diff
@@ -152,11 +167,21 @@ await using (EventHubProducer producer = client.CreateProducer())
         ///   The implementations found in this sample are synchronous and simply return a <see cref="Task.CompletedTask" />
         ///   to match the return type.
         /// </remarks>
+        ///
         private class SamplePartitionProcessor : BasePartitionProcessor
         {
+            /// <summary>Keeps track of the amount of active SamplePartitionProcessors.</summary>
+            private static int s_activeInstancesCount = 0;
+
             /// <summary>Keeps track of the amount of received events across all existing instances of this class.</summary>
             private static int s_totalEventsCount = 0;
 
+            /// <summary>
```

Comment:
nit: These are typically single-line for members, since they're just for general context.